app/javascript/react/screens/App/Overview/components/ScheduleMigrationModal/ScheduleMigrationModalBody.js

```diff
@@ -23,6 +23,9 @@ class ScheduleMigrationModalBody extends React.Component {
     });
 
     const picker = datetimeSelector.data('DateTimePicker');
+    if (defaultDate) {
+      picker.date(new Date(defaultDate));
+    }
```

Comment:
Have not tried it, so not sure if it'll work, but is it possible to use `bootstrap-datetimepicker`'s `defaultDate` option for this?

https://eonasdan.github.io/bootstrap-datetimepicker/Options/

Reply:
I tried that, and for some reason it didn't seem to work. I'll give it another try and share that branch if you want.

Reply:
Did some experimenting.  You have to also pass `useCurrent: false`

This might end up being more trouble than it's worth

Reply:
Strange, because the docs say `defaultDate` overrides `useCurrent`. Well, it works, so I'll push another commit here to use it. Thanks for experimenting!